src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxTreeFactoryService.cs

```diff
@@ -44,10 +44,10 @@ public override ParseOptions GetDefaultParseOptionsWithLatestLanguageVersion()
                 return _parseOptionWithLatestLanguageVersion;
             }
 
-            public override SyntaxTree CreateSyntaxTree(string fileName, ParseOptions options, Encoding encoding, SyntaxNode root)
+            public override SyntaxTree CreateSyntaxTree(string filePath, ParseOptions options, Encoding encoding, SyntaxNode root, ImmutableDictionary<string, ReportDiagnostic> treeDiagnosticReportingOptionsOpt)
```

Comment:
`filePath` was already the name of the parameter in the base type, but was never applied to derived types.